tests/driver_mrf24j40/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = driver_mrf24j40
```

Comment:
(also, if all other drivers are able to send via IP and not via `mrf24j40` it is safe to assume that device is at fault ;-))